tutorials/plot_background_filtering.py

```diff
@@ -483,15 +524,232 @@ def plot_signal(x, offset):
 plt.show()
 
 ###############################################################################
-# Filtering in MNE-Python
-# =======================
+# Some pitfalls of filtering
+# ==========================
+# Multiple recent papers have noted potential risks of drawing
+# errant inferences due to misapplication of filters.
+#
+# Low-pass problems
+# -----------------
+# Filters in general, especially those that are acausal (zero-phase), can make
+# activity appear to occur earlier or later than it truly did. As
+# mentioned in VanRullen 2011 [3]_, investigations of commonly (at the time)
+# used low-pass filters created artifacts. However, such deleterious effects
```

Comment:
No, I don't have a figure showing this type of distortion mostly because I added `firls` in SciPy only in 0.18, which CircleCI can't use because it will create problems with Mayavi IIRC. I also don't think we really need to, since we don't use `firls` and neither does EEGLAB anymore.